packages/near-contract-standards/src/non_fungible_token/events.ts

```diff
@@ -16,17 +16,22 @@
 import { AccountId } from "near-sdk-js";
 import { NearEvent } from "../event";
 import { TokenId } from "./token";
+import { toSnakeCase } from "../util";
 
 export type Nep171EventKind = NftMint[] | NftTransfer[] | NftBurn[];
```

Comment:
can you also add the event typf of `ContractMetadataUpdate`: https://nomicon.io/Standards/Tokens/NonFungibleToken/Event#interface